projects/layout/README.md

```diff
@@ -32,13 +36,43 @@ import { NgxConfigurableLayoutComponent, NgxConfigurableLayoutItemComponent } fr
 ```
 
 ```html
-<ngx-configurable-layout [keys]="['second-item', 'first-item']">
+<ngx-configurable-layout [keys]="[['second-item', 'first-item'], ['third-item']]">
 	<ngx-configurable-layout-item key="first-item">
-		<p>This is the first item in the template.</p>
+		<p>This is the second item in the template.</p>
```

Comment:
This is still the first element in the `template`. It is the second item in the `DOM`, though 🙂